packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -426,7 +433,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
     }
     final double vx = velocity.pixelsPerSecond.dx;
     final double vy = velocity.pixelsPerSecond.dy;
-    DismissDirection? flingDirection;
+    DismissDirection flingDirection;
```

Comment:
It seems like this should be a `late` variable however the analyzer isn't complaining, so I suppose the `late` qualifier isn't needed.